e2e/testcases/cli_test.go

```diff
@@ -1200,3 +1200,39 @@ func TestNomosStatusNameFilter(t *testing.T) {
 		nt.T.Fatalf("Expected to not find bookinfo:crontab-sync component in output:\n%s\n", string(out4))
 	}
 }
+
+func TestNomosStatusThrottling(t *testing.T) {
```

Comment:
Is this a copy of the CRD stress test with nomos status after?
Do we need 100 CRDs for throttling to happen?

Reply:
throttling seem to happen when there's a lot of CRDs in the cluster. This is reproducible with installing KCC on the cluster (~200 CRDs) or running the `TestStress100CRDs`. This e2e test is essentially running `TestStress100CRDs` and validate the output of `nomos status`.

Reply:
the test has been moved to be a part of `TestStress100CRDs`